wolfcrypt/src/sha.c

```diff
@@ -865,6 +870,13 @@ void wc_ShaFree(wc_Sha* sha)
 #ifdef WOLFSSL_IMXRT_DCP
     DCPShaFree(sha);
 #endif
+#if defined(WOLFSSL_RENESAS_RX64_HASH)
+    wolfssl_RX64_HW_Hash* hw_sha = (wolfssl_RX64_HW_Hash*)sha;
```

Comment:
Please fix mixed variable declaration. The variable needs to be declared at top of function or in a brace `{}` section.